### PR TITLE
ath79: add Support for the TP-LINK WBS510 v1.0/v2.0 Device 

### PR DESCRIPTION
**Pull request by apcameron**
This adds support for a popular low-cost 5GHz N based AP

Specifications:
- SoC: Atheros AR9344
- RAM: 64MB
- Storage: 8 MB SPI NOR
- Wireless: 5GHz 300 Mbps, 2x RP-SMA connector, 27 dBm TX power
- Ethernet: 1x 10/100 Mbps with 24V POE IN, 1x 10/100 Mbps

Installation:
Flash factory image through stock firmware WEB UI
or through TFTP
To get to TFTP recovery just hold reset button while powering on for
around 4-5 seconds and release.
Rename factory image to recovery.bin
Stock TFTP server IP:192.168.0.100
Stock device TFTP adress:192.168.0.254

Signed-off-by: Andrew Cameron apcameron@softhome.net

[wbs510v2_dmesg.txt](https://github.com/openwrt/openwrt/files/3861372/wbs510v2_dmesg.txt)
[wbs510v1_dmesg.txt](https://github.com/openwrt/openwrt/files/3861373/wbs510v1_dmesg.txt)

### CODE REVIEW

**Comment by adrianschmutzler** [comment]
Hi, I've added your changes to my staging tree
https://git.openwrt.org/?p=openwrt/staging/adrian.git;a=shortlog;h=refs/heads/staging
with the following adjustments:
- Slightly changed commit title
- Fixed bracket around mail address in Signed-off
- Fixed SUPPORTED_DEVICES for wbs510-v1 (wbs510-v1 to wbs510)

Please have a look whether you are fine with the current state.

**Comment by apcameron** [comment]
The version in your staging tree looks good to me.

**Comment by apcameron** [comment]
I think I have spotted a Problem with the v2 Device and will need to update that commit

**Comment by apcameron** [comment]
Updated to correct the SupportList entries in the V2 Device

**Comment by apcameron** [comment]
Please update your staging tree based on the updates to my commit to ath79: add Support for the TP-LINK WBS510 V2.0 Device

**Comment by adrianschmutzler** [comment]
Thanks for the update. If I need to update my staging tree anyway, maybe include the following changes into your version first:
- Change commit titles as in my current staging
- Update Signed-off (brackets) for V1 commit
- Update SUPPORTED_DEVICES for v1

**Comment by apcameron** [comment]
@adrianschmutzler I have made the changes you requested and it is ready for you to review/update in your staging area.

File: tools/firmware-utils/src/tplink-safeloader.c [hunk 1]
```diff
@@ -545,7 +545,8 @@ static struct device_info boards[] = {
 			"SupportList:\r\n"
 			"WBS510(TP-LINK|UN|N300-5):1.20\r\n"
 			"WBS510(TP-LINK|US|N300-5):1.20\r\n"
-			"WBS510(TP-LINK|EU|N300-5):1.20\r\n",
+			"WBS510(TP-LINK|EU|N300-5):1.20\r\n"
+			"WBS510(TP-LINK|CA|N300-5):1.20\r\n",
```
Comment (adrianschmutzler) [inline tools/firmware-utils/src/tplink-safeloader.c]: So, how sure are you about the "1.20" here?

**Comment by adrianschmutzler** [comment]
Meanwhile, I pushed the updated commits to my staging tree.

File: tools/firmware-utils/src/tplink-safeloader.c (hunk 1 above)
Comment (apcameron) [reply tools/firmware-utils/src/tplink-safeloader.c]: Yes in this case the 1.20 is correct.
I just added the Canadian Model

**Comment by adrianschmutzler** [comment]
Merged, thanks.

**Pull request closed without merging**